app/src/main/res/values/strings.xml

```diff
@@ -383,6 +383,8 @@
     <string name="language">Language</string>
     <string name="choose_language">Choose your language</string>
     <string name="notification">Notifications</string>
+    <string name="base_url">Base Url</string>
```

Comment:
It should be `Base URL`

Reply:
@luckyman20 I have used "Base Url" because the Title also shows "Base Url". It's better to have consistency in messages.

Reply:
@miPlodder I think you can change that too also because URL is a short form for Uniform Resource Locater & short forms should be written in caps only.